infra/e2e/turing.values.remote.yaml

```diff
@@ -4,10 +4,33 @@ turing:
 
   config:
     ClusterConfig:
-      VaultConfig:
-        Address: http://vault:8200
-        Token: root
+      EnsemblingServiceK8sConfig: {}
 
 merlin:
   vault:
-    enabled: false
\ No newline at end of file
+    enabled: false
+  environmentConfigs:
```

Comment:
We already have the `environmentConfigs` in `infra/charts/turing/values.yaml` - why is it repeated here now?

Reply:
This is referenced in https://github.com/shydefoo/turing/blob/0ca6c7003daed4cb066076fbff469e25c0f439d7/.github/actions/setup-test-cluster/action.yaml#L194, to make sure the value `.turing.environmentConfigs.[0].name`  matches `.merlin.environmentConfigs.[0].name`